package-res/reportviewer/reportviewer.js

```diff
@@ -826,12 +830,36 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
           dlg.showBackgroundBtn(_Messages.getString('FeedbackScreenBackground'));
         }else {
           scheduleScreenBtnCallbacks.shift();
+          dlg.hideBackgroundBtn();
         }
 
         dlg.callbacks = scheduleScreenBtnCallbacks;
         return dlg;
       },
 
+      renderBackgroundBtnListener: function (editMode) {
```

Comment:
Just one change: "renderBackgroundBtnListener" is a horrible name for an event handler. functions starting with a verb normally signify an normal method, so here I would expect that something is rendered. 

Please name it: editModeToggledHandler - this makes it clear that it is an event handler, and what event it handled.

And change the name of "dlg2" to "feedbackScreen" (again, readability - this way you can look at the variable and see instantly what it is. "dlg2" is meaningless compared to "feedbackScreen".